### PR TITLE
Lets BoH hold mech parts

### PR DESCRIPTION
**Pull request by Bawhoppen**
> couldn't even hold your ultra ac2 in your bag of holding

### CODE REVIEW

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 1]
```diff
@@ -35,7 +35,7 @@
 	desc = "A backpack that opens into a localized pocket of Blue Space."
 	origin_tech = "bluespace=4"
 	icon_state = "holdingpack"
-	max_w_class = 5
+	max_w_class = 10
 	max_combined_w_class = 35
```
Comment (ChangelingRain) [inline code/game/objects/items/weapons/storage/backpack.dm]: 210

Comment (Bawhoppen) [inline code/game/objects/items/weapons/storage/backpack.dm]: what

Comment (ChangelingRain) [inline code/game/objects/items/weapons/storage/backpack.dm]: slots*max_w_class

Comment (KazeEspada) [inline code/game/objects/items/weapons/storage/backpack.dm]: Not the way storage works.

Comment (Bawhoppen) [inline code/game/objects/items/weapons/storage/backpack.dm]: why

**Comment by PKPenguin321** [comment]
>he thinks BoHs are worthless

w e w

File: code/game/objects/items/weapons/storage/backpack.dm (hunk 1 above)
Comment (KazeEspada) [inline code/game/objects/items/weapons/storage/backpack.dm]: It makes slots until it reaches 35 w class

Comment (Bawhoppen) [inline code/game/objects/items/weapons/storage/backpack.dm]: yes so? if you want to hold like 3 mech guns in your boh you can but no more than that

Comment (xxalpha) [inline code/game/objects/items/weapons/storage/backpack.dm]: That defeats the purpose, though.

Comment (Bawhoppen) [inline code/game/objects/items/weapons/storage/backpack.dm]: you can still hold the same amount of normal stuff, now you can optionally carry a mech thing too if you are so inclined

**Comment by MMMiracles** [comment]
bags that let you create singularites on the fly and store hardsuits are useless.

yes it all makes sense now.

**Comment by Bawhoppen** [comment]
why are you guys micro-examining a dumb joke

File: code/game/objects/items/weapons/storage/backpack.dm [hunk 2]
```diff
@@ -35,7 +35,7 @@
 	desc = "A backpack that opens into a localized pocket of Blue Space."
 	origin_tech = "bluespace=4"
 	icon_state = "holdingpack"
-	max_w_class = 5
+	max_w_class = 10
```
Comment (phil235) [inline code/game/objects/items/weapons/storage/backpack.dm]: there's no such thing as a 10 w_class, make it 6 instead.

Comment (Bawhoppen) [inline code/game/objects/items/weapons/storage/backpack.dm]: won't 10 default to 6 then
whatever I'll change it

**Comment by KazeEspada** [comment]
Phil, he was future proofing it, because for some reason, coders keep making bigger and bigger items. There are some W_class 6 items in lavaland for instance.

**Comment by KorPhaeron** [comment]
Whatever

**Comment by Bawhoppen** [comment]
why the fuck you rename this

**Pull request merged**